modules/security-context-constraints-example.adoc

```diff
@@ -82,6 +82,8 @@ By default, cluster administrators, nodes, and the build controller are granted
 access to the privileged SCC. All authenticated users are granted access to the
 restricted SCC.
 
+// TODO: Does this statement need to be updated? ^ Do we need to check the examples in this file?
```

Comment:
@stlaz Does the above paragraph need updated to say restricted-v2?

```
By default, cluster administrators, nodes, and the build controller are granted
access to the privileged SCC. All authenticated users are granted access to the
`restricted-v2` SCC.
```

Reply:
Yes, that should be changed